lib/validators/specResolver.ts

```diff
@@ -1027,4 +1033,25 @@ export class SpecResolver {
     }
     return result
   }
+
+  /**
+   * Check if exist undefined within-document reference
+   */
+  private checkWithInReferences() {
```

Comment:
Can we change the name to `VerifyInternalReference`?

Reply:
fixed it